StripeIdentity/StripeIdentity/Source/NativeComponents/Coordinators/VerificationSheetController.swift

```diff
@@ -143,6 +146,7 @@ final class VerificationSheetController: VerificationSheetControllerProtocol {
             self?.verificationPageResponse = result
             if case .success(let verificationPage) = result {
                 self?.startLoadingMLModels(from: verificationPage)
+                self?.isVerificationPageSubmitted = verificationPage.submitted
```

Comment:
nit: usually referencing self weakly more than once is not recommended. As technically self can get deallocated in-between first and second calls. You can add `guard let self = self else { return }` and then use self for both lines